docs/team/hakujitsu.md

```diff
@@ -3,32 +3,81 @@ layout: page
+     functionalities belonging to Person but also additional features, such as the sale breakdown function.
+     * It affected commands to be added in future, namely the command to view sales statistics. 
+     * It required an in-depth analysis of design alternatives, as the relationship between Person and Sale needed to be considered carefully.
+ 
+ New Feature: Ad Hoc Graphical User Interface (GUI)
+ * What it does: The Ad Hoc GUI can display either the Sales Panel or Tag Panel, 
+ toggling between both depending on which command was most recently executed. 
+ For example, upon executing a Sale command, the Sale Panel will be displayed, and vice versa with the Tag Panel.
+ * Justification: This feature reduces the number of panels needed to display the different models.
+ It also allows for relevant information to be shown after every command execution, improving the general user experience.
+ * Highlights: This feature required changes to the existing CommandResult class, and affected existing Command classes for Sale and Tag.
+ 
+ Enhancement: Addition of Remark field to Person class
+ * What it does: allows the user to add miscellaneous remarks to contacts in the address book. 
+ * Justification: allows for the user to store information that does not correspond to any particular person field.
+ For example, they can use it to take note of the client's needs, wants and preferences.
+ * Highlights: This was a relatively simple feature to implement.
+ 
+ ### Contributions to documentation
+ 
+ - User guide:
+     - Wrote the section to explain the user interface.
+     - Wrote the sections for `sale list`, `sale add`, `sale delete`, `sale edit` and `sale breakdown`commands in the User Guide.
+ - Developer guide: 
+     - Wrote the sections for the implementation and use cases related to the 'Sale' feature in the Developer Guide,
+     and added all UML diagrams seen in those sections.
+ 
+ ### Contributions to team-based tasks :
+ 
+ * Handled release management for an informal milestone set by the team, v1.3a, intended to be a checkpoint within v1.3.
+ * Handled the JAR release in Week 10.
+ * Maintained the issue tracker by removing duplicate issues after the conclusion of the mock PE. 
+ * Wrote an explanation of the user interface in the User Guide.
+ * Added the parameter tables for Person related commands.
+ * Update DeleteSequenceDiagram with updated implementation.
+ * Managed the user stories section of the Developer Guide.
+ 
+ ### Review/mentoring contributions: Links to PRs reviewed, instances of helping team members in other ways
+  Here is a list of PRs reviewed by me with non-trivial comments: 
+  [#68](https://github.com/AY2021S1-CS2103T-T11-1/tp/pull/68), 
+  [#77](https://github.com/AY2021S1-CS2103T-T11-1/tp/pull/77), 
+  [#148](https://github.com/AY2021S1-CS2103T-T11-1/tp/pull/148), 
+  [#144](https://github.com/AY2021S1-CS2103T-T11-1/tp/pull/114), 
```

Comment:
There seems to be a typo here. The text says 144 but the link points to 114

Reply:
Thanks, just fixed it!